packages/model-viewer/src/three-components/gltf-instance/correlated-scene-graph.ts

```diff
@@ -123,20 +123,38 @@ export class CorrelatedSceneGraph {
 
     const defaultMaterial = {name: 'Default'} as Material;
     const defaultReference = {type: 'materials', index: -1} as GLTFReference;
-    if (cloneGLTF.materials == null) {
-      cloneGLTF.materials = [];
-    }
-    defaultReference.index = cloneGLTF.materials.length;
-    cloneGLTF.materials.push(defaultMaterial);
 
     for (let i = 0; i < originalThreeGLTF.scenes.length; i++) {
       this[$parallelTraverseThreeScene](
           originalThreeGLTF.scenes[i],
           cloneThreeGLTF.scenes[i],
           (object: ThreeSceneObject, cloneObject: ThreeSceneObject) => {
-            const elementReference =
+            let elementReference =
                 upstreamCorrelatedSceneGraph.threeObjectMap.get(object);
 
+            if (((object as Mesh).isMesh || (object as Material).isMaterial) &&
+                elementReference == null) {
+              // Checks if default material was allready addded to the gltf.
+              if (cloneGLTF.materials) {
+                const material =
+                    cloneGLTF.materials[cloneGLTF.materials.length - 1];
+                if (material.name === 'Default') {
```

Comment:
This prevents a Default material from being added twice, the first time it's added is during the original GLTF correlation step.